.travis.yml

```diff
@@ -11,3 +11,17 @@ script:
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+
+deploy:
+  # deploy release
+  - provider: script
+    script: ./gradlew bintrayUpload install -PdryRun=false
+    on:
+      tags: true
+      condition: "$TRAVIS_TAG =~ ^Release.*$"
```

Comment:
`TRAVIS_TAG` it`s a Git tag? and could you please put an example of a valid tag that triggers this deploy?

Reply:
changing this, that won't trigger, I was taking the release name on Github as the tag name